content/docs/reference/shared-secret.mdx

```diff
@@ -1,8 +1,7 @@
 ---
-id: shared-secret
-title: Shared Secret
-description: |
-  Shared Secret is the base64 encoded 256-bit key used to mutually authenticate requests between services.
+id: shared-secret-settings
+title: Shared Secret Settings
```

Comment:
If it were up to me I would keep the existing title and URL path here. This is really just two ways to configure the same setting.

```suggestion
id: shared-secret
title: Shared Secret
```

And likewise for signing key.